src/app/api/gql/fragments/events/fragment-add-push-source.graphql

```diff
@@ -0,0 +1,93 @@
+fragment AddPushSourceEvent on AddPushSource {
+  sourceName
+  read {
```

Comment:
There is large duplication with fragment for `SetPollingSource`. Extract common definitions for steps and reuse in both events.

Reply:
Fixed